benchmarks/xlmr/ootb/xlmr_utils.py

```diff
@@ -0,0 +1,47 @@
+
+import numpy as np
+
+"""
+Util functions for xlmr benchmark
```

Comment:
Nit: XLMR or XLM-R?

Reply:
Good call. I use these pretty interchangeably. XLM-R is more accurate so I'll keep to that in comments and xlmr in code.